torch/_inductor/codegen/triton.py

```diff
@@ -1784,7 +1785,7 @@ def codegen_kernel(self, name=None):
         code = IndentedBuffer()
 
         size_hints = [
-            next_power_of_2(V.graph.sizevars.size_hint(numel)) for numel in self.numels
+            next_power_of_2(V.graph.sizevars.size_hint(numel)) if not free_symbols(numel) else 64 for numel in self.numels
```

Comment:
how did you choose 64 here? 

We should still be using the size_hint of the numel if it exists. 

We could also check if the symbols in the numel have bounds on them, and do something with that.

Reply:
thanks! just realized it's a relic and we don't need this hardcoded hint anymore.

Reply:
discussed with Elias offline, and added `shape_env.var_to_size_hint` to handle this